src/components/Modal.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+
+
+class Modal extends Component{
+    render(){
```

Comment:
same ca si la state properties, it's cleaner daca extragi proprietatiile din this.props la inceput de functie